content/source/docs/enterprise/system-overview/security-model.html.md

```diff
@@ -0,0 +1,100 @@
+
+Additionally, we recommend restricting access to the nodes that are running TFE. TFE can not ensure the security or integrity of its data if the underlying infrastructure is compromised.
+
+### Enable Optional Security Features
+
+Once you are ready to use Terraform Enterprise for production workloads, we recommend enabling these optional security features.
+
+#### Strict Transport Security Header
+
+You can configure TFE to set the [Strict Transport Security (HSTS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security) header via the Settings page of the installer dashboard by enabling “Force TLS” radio button under “SSL/TLS Configuration”
+
+You can also enable this setting via the application settings file file with the [force_tls](../install/automating-the-installer.html#force_tls) setting.
+
+~> **Note:** Once properly configured, the HSTS header cannot be disabled and will prevent clients from accessing your TFE domain via HTTP or HTTPS using a self-signed cert. We recommend only enabling this setting for production TFE deployments.
+
+#### Restrict Terraform Build Worker Metadata Access
+
+By default Terraform Enterprise does not prevent Terraform operations from accessing the instance metadata service, which may contain IAM credentials or other sensitive data. Refer to [AWS](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html), [Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/instance-metadata-service?tabs=windows), or [Google Cloud's](https://cloud.google.com/compute/docs/storing-retrieving-metadata) documentation for more information on this service.
+
+TFE allows you to restrict access to the metadata endpoint from Terraform operations, preventing workspaces from reading any data from the metadata service. This feature can be enabled via the “Restrict Terraform Build Worker Instance Metadata Access” setting under the “Advanced Configuration” section in the installer dashboard’s settings page, or via the [restrict_worker_metadata_access](../install/automating-the-installer.html#restrict_worker_metadata_access) setting in the application settings file.
+
+Unless you are relying on the [instance profile as a means of providing default credentials to TFE workspaces](../before-installing/index.html#aws-specific-configuration), we recommend enabling this setting to prevent Terraform operations from accessing the instance metadata endpoint. 
+
+#### Ensure Global Remote State Sharing is Disabled
+
+TFE allows administrators to enable [global remote state sharing](../admin/general.html#remote-state-sharing), a feature that allows any workspace to access the state versions of any other workspace within the same organization. We recommend disabling the feature and relying on [controlled remote state access](https://www.hashicorp.com/blog/announcing-controlled-remote-state-access-for-terraform-cloud-and-enterprise) if you need to share state between workspaces.
```

Comment:
Also - wondering which type of admin - there are two listed above. Maybe we should specify if it's both or just one. 
```suggestion
Terraform Enterprise allows administrators to enable [global remote state sharing](../admin/general.html#remote-state-sharing), which allows any workspace to access the state versions of any other workspace within the same organization. We recommend disabling this feature and relying on [controlled remote state access](https://www.hashicorp.com/blog/announcing-controlled-remote-state-access-for-terraform-cloud-and-enterprise) if you need to share state between workspaces.
```

Reply:
I'm referring to site admins here. I will update that to be more clear